packages/ui-components/src/components/tree-item/tree-item.base.scss

```diff
@@ -8,6 +8,7 @@
 	 * @prop --children-offset: Offset of the child nodes in px.
 
 	 * @prop --connector-lines-color: Line color of children's connectors.
+	 * @prop --vertical-lines-height: Line height of children's connectors.
```

Comment:
I think it's missing the documentation for the other added CSS props. Not sure if it was intended.